examples/running_average/client/running_average.cc

```diff
@@ -84,9 +86,11 @@ int main(int argc, char** argv) {
   // Retrieve average.
   int average_0 = retrieve_average(stub_0.get());
   LOG(INFO) << "client 0 average: " << average_0;
+  assert(average_0 == 100);
```

Comment:
This looks like an unrelated change?

Reply:
Moved to https://github.com/project-oak/oak/pull/1006